validators/crossingBuildings/map.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+var turf = require('turf');
+var _ = require('underscore');
+var rbush = require('rbush');
+
+module.exports = function(tileLayers, tile, writeData, done) {
+  var layer = tileLayers.osm.osm;
+  var buildings = {};
+  var bboxes = [];
+  layer.features.forEach(function(val) {
+    if (val.geometry.type === 'Polygon' && val.geometry.coordinates.length === 1 && val.properties.building === 'yes') {
```

Comment:
hey @geohacker  because there is problems with self-intersect Polygons https://github.com/osmlab/osmlint/issues/29, for now is better avoid those buildings.